conf/resources/data/etmp.eligibility.SIA/eligibilityPass.json

```diff
@@ -61,10 +61,10 @@
 
   "charges": [
     {
-      "chargeType": "VAT Mig Return Debit Charge",
-      "mainType": "VAT Mig Return Charge",
-      "mainTrans": "7700",
-      "subTrans": "1174",
+      "chargeType": "In Year RTI Charge - Tax",
+      "mainType": "In Year RTI Charge (FPS)",
+      "mainTrans": "4920",
```

Comment:
Should we use the SIA 4530, 1000 charge?

Reply:
Spoke to Robyn and @SpencerCGriffiths yesterday. they're "pointing to SA charges for now until SIA implementation ticket".